docker-hub/new-company.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Create a company
+description: Learn how to create a company.
+keywords: company, hub, organization, company owner
+---
+
+## Create a company
+
+To create a new company:
+
+1. In Docker Hub, navigate to an organization you want to place under a company. The organization must have a Business subscription.
```

Comment:
Could something like this sound more natural?

```suggestion
1. In Docker Hub, navigate to an organization on a business subscription that want to place under a company.
```

Reply:
I personally think it's ok. I think it reads clearer if it is two sentences. This PR will be checked by my docs-team colleagues and if they pick up on it too, I'll reassess.